### ISSUE TITLE
ICANN verification?

### ISSUE THREAD

**Issue by alysonann** [susanBuck/e2-fall20#47]
Yesterday, my site was working properly [http://e2p1.alysonann.bike/](http://e2p1.alysonann.bike/), and tonight I am getting this upsetting message.

<img width="1068" alt="Screen Shot 2020-10-01 at 6 34 11 PM" src="https://user-images.githubusercontent.com/31620333/94870097-b8219e00-0414-11eb-8e12-c434456255e8.png">

My domain is on name.com, and I contacted their chat support. They confirmed that verification with them is all set, and I should try contacting ICANN. (Indeed, [http://alysonann.bike/](http://alysonann.bike/) does work.) I'm about to try and do that (although I'm assuming fast support isn't going to happen), but I thought I would throw up a post and see if anyone was familiar with this problem?

**Comment by susanBuck** [comment]
Hi @alysonann - When did you complete the verification with name.com? If it was done recently, perhaps it is still within the 24-48 hour wait period they note:

> We estimate it may take 24 to 48 hours for the website to come back online.

As for how it relates to your p1 submission - don't stress about that. Because this is beyond your control and not related to any of the setup procedures we've covered in this course, it won't impact your grade. If it's still not working on production when I get to reviewing your project, I can always clone locally to view your work.

Knowing this, let's give it 24 hours and see if it resolves itself (or you hear back from ICANN). If that doesn't resolve things, we can look at next steps.

**Comment by alysonann** [comment]
I just completed the verification today when I saw the ICANN error. I never received any emails from name.com until today, which I'm guessing is the root of the issue! (My email is correct now, but I do remember fixing a typo in my email immediately after I made the account a few weeks ago.) Since name.com's customer service has been a mess (apparently the only thing ICANN can do is help me file a complaint against name.com, so name.com really shouldn't have suggested I contact them), I went ahead and set up a site with a different domain provider which seems to be working (and sent me a verification email), so I will go with that. Thanks for your help!
